extensions/blocks/map/deprecated/v1/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import save from './save';
+
+const attributes = {
+	align: {
+		type: 'string',
+	},
+	points: {
+		type: 'array',
+		default: [],
+	},
+	mapDetails: {
+		type: 'boolean',
+		default: true,
+	},
+	zoom: {
+		type: 'integer',
+		default: 13,
+	},
+	mapCenter: {
+		type: 'object',
+		default: {
+			longitude: -122.41941550000001,
+			latitude: 37.7749295,
+		},
+	},
+	mapStyle: {
+		type: 'string',
+		default: 'default',
+	},
+	markerColor: {
+		type: 'string',
+		default: 'red',
+	},
+	preview: {
+		type: 'boolean',
+		default: false,
+	},
+	scrollToZoom: {
+		type: 'boolean',
+		default: false,
+	},
+	mapHeight: {
+		type: 'integer',
+	},
+	showFullscreenButton: {
+		type: 'boolean',
+		default: true,
+	},
+};
+
+export default {
+	attributes,
+	migrate: oldAttributes => {
+		const className =
+			oldAttributes.className.replace( /is-style-[^ ]+/, '' ) +
```

Comment:
I am getting a blank screen when trying to edit an old post that has 2 maps with custom styles, once I have switched to this branch:

![image](https://user-images.githubusercontent.com/426388/77451216-ac657700-6df4-11ea-8c5a-870c11025868.png)

It seems to happen here during the migration.

Reply:
Fixed in 285323a0d.